pom.xml

```diff
@@ -146,6 +147,14 @@
         <mockito.version>3.4.6</mockito.version>
         <mockito-scala.version>1.17.14</mockito-scala.version>
         <assertj.version>3.23.1</assertj.version>
+        <testcontainer.version>1.19.3</testcontainer.version>
+        <zookeeper.version>3.8.0</zookeeper.version>
```

Comment:
Please make sure to declare these dependencies in the [LICENSE](https://github.com/apache/incubator-streampark/blob/dev/dist-material/release-docs/LICENSE)

Reply:
These two dependency only using in test scope. So it doesn't package into dist just like mysql.